CosmicReco/src/PDFFit.cc

```diff
@@ -1,359 +1,443 @@
-	double pdf_val = pdf_val0*(1-t_d) + pdf_val1*t_d;
+  double pdf_val00 = pdf_val000 * (1 - s_d) + pdf_val100 * s_d;
+  double pdf_val01 = pdf_val001 * (1 - s_d) + pdf_val101 * s_d;
+  double pdf_val10 = pdf_val010 * (1 - s_d) + pdf_val110 * s_d;
+  double pdf_val11 = pdf_val011 * (1 - s_d) + pdf_val111 * s_d;
+  double pdf_val0 = pdf_val00 * (1 - tau_d) + pdf_val10 * tau_d;
+  double pdf_val1 = pdf_val01 * (1 - tau_d) + pdf_val11 * tau_d;
+  double pdf_val = pdf_val0 * (1 - t_d) + pdf_val1 * t_d;
 
-	return pdf_val;
+  return pdf_val;
+}
 
+// This 3 functions talk to the drift util:
+double GaussianPDFFit::calculate_DOCA(ComboHit const& chit, double const& a0, double const& a1,
+                                      double const& b0, double const& b1,
+                                      const Tracker* tracker) const {
+  double doca = DriftFitUtils::GetTestDOCA(chit, a0, a1, b0, b1, tracker);
+  return (doca);
 }
 
-    // This 3 functions talk to the drift util:
-    double GaussianPDFFit::calculate_DOCA(ComboHit const& chit, double a0, double a1, double b0, double b1, const Tracker* tracker)const{
-	double doca = DriftFitUtils::GetTestDOCA(chit, a0,a1,b0,b1, tracker); 
-	return (doca);
+double GaussianPDFFit::calculate_ambig(ComboHit const& chit, double const& a0, double const& a1,
+                                       double const& b0, double const& b1,
+                                       const Tracker* tracker) const {
+  double ambig = DriftFitUtils::GetAmbig(chit, a0, a1, b0, b1, tracker);
+  return (ambig);
 }
 
-    double GaussianPDFFit::calculate_ambig(ComboHit const& chit, double a0, double a1, double b0, double b1, const Tracker* tracker)const{
-	double ambig = DriftFitUtils::GetAmbig(chit, a0,a1,b0,b1, tracker); 
-	return (ambig);
+double GaussianPDFFit::TimeResidual(double const& doca, StrawResponse const& srep, double const& t0,
+                                    ComboHit const& hit, const Tracker* tracker) const {
+  double tres = DriftFitUtils::TimeResidual(doca, srep, t0, hit, tracker);
+  return (tres);
 }
 
-    double GaussianPDFFit::TimeResidual(double doca, StrawResponse const& srep , double t0 ,  ComboHit const& hit, const Tracker* tracker)const{
-	double tres =  DriftFitUtils::TimeResidual(doca, srep, t0, hit, tracker);
-	return (tres);
+double GaussianPDFFit::operator()(const std::vector<double>& x) const {
+
+  double const& a0 = x[0];
+  double const& a1 = x[1];
+  double const& b0 = x[2];
+  double const& b1 = x[3];
+  double t0 = x[4];
+  long double llike = 0;
+
+  for (size_t i = 0; i < this->chits.size(); i++) {
+    double doca = calculate_DOCA(this->chits[i], a0, a1, b0, b1, this->tracker);
+    double time_residual = this->TimeResidual(doca, this->srep, t0, this->chits[i], this->tracker);
+    double pdf_t = 1 / sqrt(2 * TMath::Pi() * this->sigma_t * this->sigma_t) *
+                   exp(-(time_residual * time_residual) / (2 * this->sigma_t * this->sigma_t));
```

Comment:
I'm going to suggest that going forward we have any large scale reformatting put in separate pull requests - it just makes a review of this much code harder